compiler/optimizer/VPHandlers.cpp

```diff
@@ -93,6 +93,42 @@
 extern TR::Node *constrainChildren(OMR::ValuePropagation *vp, TR::Node *node);
 extern TR::Node *constrainVcall(OMR::ValuePropagation *vp, TR::Node *node);
 
+static bool canTrustAsFixedClass(OMR::ValuePropagation *vp, TR::SymbolReference *symRef, TR_OpaqueClassBlock *classObject)
```

Comment:
I think this function could have `TR::Compilation *` as a parameter instead of `OMR::ValuePropagation *`.

Reply:
Addressed in [c51517f](https://github.com/eclipse-omr/omr/compare/2cb762122810acf1b3e93226839c6d6032c9a475..c51517f9015c5666537d1e78ef3d08501bf49291)